api/tests/routes/native/v1/account_test.py

```diff
@@ -441,8 +439,8 @@ def test_account_creation_performance_tests(self, mocked_check_recaptcha_token_i
 
         user = users_models.User.query.first()
         assert (
-            users_models.Token.query.filter_by(user=user).first().value
-            == f"performance-tests_email-validation_{user.id}"
+            token_utils.Token._get_redis_key(token_utils.TokenType.EMAIL_VALIDATION, user.id)
+            == f"pcapi:token:email_validation_{user.id}"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ça ressemble à un breaking change ça :thinking: 
Comment ça se fait qu'on ait ce changement ?

Reply:
Update: pas de soucis, c'est obsolète et à supprimer :heavy_check_mark: